tests/integration/golang/aim/experiment/get_experiments_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build integration
+
+package experiment
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetExperimentsTestSuite struct {
+	suite.Suite
+	client   *helpers.HttpClient
+	fixtures *fixtures.ExperimentFixtures
+}
+
+func TestGetExperimentsTestSuite(t *testing.T) {
+	suite.Run(t, new(GetExperimentTestSuite))
+}
+
+func (s *GetExperimentsTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(helpers.GetServiceUri())
+
+	fixtures, err := fixtures.NewExperimentFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.fixtures = fixtures
+}
+
+func (s *GetExperimentsTestSuite) Test_Ok() {
+	experiments, err := s.fixtures.CreateExperiments(context.Background(), 5)
+	assert.Nil(s.T(), err)
+	defer func() {
+		assert.Nil(s.T(), s.fixtures.UnloadFixtures())
+	}()
+	var resp []response.GetExperiment
+
+	err = s.client.DoGetRequest(
+		fmt.Sprintf(
+			"/experiments/",
+		),
+		&resp,
+	)
+	assert.Nil(s.T(), err)
+
+	for idx := 0; idx < 5; idx++ {
```

Comment:
Im not fully sure that this is correct. If endpoint will give 6 records instead of 5, but you iterate only over 5 so it will be a mistake in endpoint logic. So will be nice to compare the length of response. Also, you can iterate over `resp` directly and compare with what you have in `runs`, so you will always make sure that number of returned records is the same with what you've created.